### PR TITLE
enabled syncd-rpc container swap for PFC Asym tests

### PR DESCRIPTION
**Pull request by antonptashnik**
### Description of PR

Summary: enabled syncd-rpc container swap for PFC Asym tests
Fixes # (issue)
PFC Asym tests require syncd-rpc container to read port/queue counters. Made the container to be installed for duration of the tests

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
Let PFC Asym tests read packet counters
#### How did you do it?
Added syncd-rpc installation on test start
#### How did you verify/test it?
py.test --inventory=../ansible/lab,../ansible/veos --testbed_file=../ansible/testbed.csv --module-path=../ansible/library -v -rA --topology=t0,any pfc_asym/test_pfc_asym.py

### CODE REVIEW

**Comment by antonptashnik** [comment]
@ZhaohuiS please review

File: tests/pfc_asym/test_pfc_asym.py [hunk 1]
```diff
@@ -5,6 +5,10 @@
     pytest.mark.topology('t0')
 ]
 
+@pytest.fixture(scope='module', autouse=True)
+def prepare_syncdrpc(swapSyncd):
+    pass
```
Comment (ZhaohuiS) [inline tests/pfc_asym/test_pfc_asym.py:10]: @antonptashnik Why define a pass function here?

Comment (antonptashnik) [reply tests/pfc_asym/test_pfc_asym.py:10]: here `prepare_syncdrpc` is defined to make `swapSyncd` to be auto-executed once before all suite tests, so in fact it does not need body. It's only use is to set scope and autouse for `swapSyncd` to be called for PFC asym test module

**Comment by antonptashnik** [comment]
@ZhaohuiS @wangxin please review

**Review by ZhaohuiS — DISMISSED** [review]
LGTM

**Review by neethajohn — APPROVED** [review]

**Review by ZhaohuiS — APPROVED** [review]

**Review by wangxin — APPROVED** [review]

**Pull request merged**